website/static/js/commentpane.js

```diff
@@ -11,7 +11,10 @@
         onOpen: function() {}
     };
 
-    var CommentPane = CommentPane || function(selector, options) {
+    /*
+       mode: 'widget', 'pane' or 'page'
+     */
+    var CommentPane = CommentPane || function(selector, hostPage, hostName, mode, options) {
```

Comment:
<code>hostPage</code> and <code>hostName</code> is useful when clicking the bar on the comment pane and **update comment count** to 0. (See line 67) These two definitely cannot be removed

<code>mode</code> is useful on pane resizing. I'm not sure I can remove that... will try.